adapters/text_embeddings/text_embeddings.py

```diff
@@ -0,0 +1,80 @@
+import openai
+from openai import NOT_GIVEN
```

Comment:
this one shouldn't be removed? the all folder?

Reply:
Done ✅